default.vcl

```diff
@@ -61,6 +61,8 @@ sub vcl_recv {
         # Routing preset here as vulcan is unable to route on query strings
     } elseif (req.url ~ "\/content\?.*isAnnotatedBy=.*") {
         set req.http.Host = "public-content-by-concept-api";
+    } elseif (req.url ~ "\/concept\/search.*$") {
```

Comment:
Reverted to simple regex, If we try to match both `/concept/search/param` and NOT match /concept/search-other/bla` it gets too complicated. If needed we can put a condition before this one in case someone names another service similarly.